docs/api/createAsyncThunk.md

```diff
@@ -246,6 +246,43 @@ const onClick = () => {
 }
 ```
 
+## Handling Thunk Errors
+
+Generally, whenever something is `throw`n during the execution of your `payloadCreator`, that will lead to a `rejected` action to be dispatched and returned from the thunk's promise, so if you are doing `await dispatch(myThunk())`, you will also get that rejected action - not, as you might expect on first thought, a Promise rejection. This is due to the fact that we want to prevent uncaught promise rejections for those who do not use the result of `dispatch`.
+
+Either way, that rejected action will have the object that was `throw`n as `action.meta.error`. However, to ensure serializability, everything that does not match the interface
+
+```ts
+export interface SerializedError {
+  name?: string
+  message?: string
+  stack?: string
+  code?: string
+}
+```
+
+will have been removed from it.
+
+If you want to pass out additional information or information in another shape, instead of `throw`ing it, you can use the `api.rejectWithValue` method instead.
```

Comment:
```suggestion
In the scenario that you're able to make use of a specific error shape with known properties in your reducers or components, you should use the `thunkAPI.rejectWithValue` method instead of `throw`ing an error.
```

Reply:
Clarity & context suggestion. We also reference `thunkAPI` everywhere else, so we should at least keep that standard.

Additional text that may be useful at the end of this: "This is particularly common when expecting field-level validation errors from an API."